test/RewriteSpec.hs

```diff
@@ -23,13 +23,13 @@ spec = do
     it "quotes an unquoted meta.homepage URL" do
```

Comment:
Might be nice to have an actual test case for running `nixpkgs-review` and getting the output into the PR comment, though that'd require some refactoring of the PR msg body to be a pure function returning `Text` instead of only being formed/called when we're actually posting.

Reply:
Yeah, I think I want to standardize the "checks" like we are standardizing the "rewrites", so I think that's the direction to go to get some testing sooner rather than later.

Reply:
There were definitely quite a few times working on nixpkgs-review where it was clunky to test the PR msg body, and I had some bugs with it; it'd be nice to get some test cases like the ones I added in `nixpkgs-review`:

https://github.com/Mic92/nixpkgs-review/blob/master/nixpkgs_review/tests/assets/expected_pr_report_1234.md
https://github.com/Mic92/nixpkgs-review/blob/master/nixpkgs_review/tests/test_report.py#L32-L35

Aside from making development and refactoring easier to test changes like this one, it also makes it easier to look at the actual msg.  If we fix https://github.com/ryantm/nixpkgs-update/issues/165 it'd be the ideal time to write a test case!